V1Michigan/v1
# Type script migration

### Status
**WIP**

### Description
Successfully migrated component code to TypeScript.


### Migrations
All component files have been migrated to TypeScript from JavaScript.

### Steps to Test or Reproduce

```sh
git pull
git checkout type-script-migration
npm run dev
```

Also make sure you have the updated `.env` with Supabase keys (in Slack).
